### DISCUSSION TITLE
What is the Unit of real acceleration, adjusted to remove gravity

### DISCUSSION THREAD

**Discussion by JuMalIO** [jrowberg/i2cdevlib#599] [Q&A]
What is the Unit of "aaReal" variable (real acceleration, adjusted to remove gravity), in this example:
https://github.com/jrowberg/i2cdevlib/blob/master/Arduino/MPU6050/examples/MPU6050_DMP6/MPU6050_DMP6.ino
line 304?

How to convert it to g-force value?

**Comment by nices0325** [comment]
really wanna know this either
